Source/UIKitExtensions.swift

```diff
@@ -57,6 +57,10 @@ public extension UINavigationController {
         }else{
             self.topViewController?.navigationItem.rightBarButtonItems = [spacer, item]
         }
+        
+        if sideMenuButtonAdded != nil {
```

Comment:
:) that's not how you do it (have a look [here](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/OptionalChaining.html))

change to `completion?(button)`